app/src/main/java/com/quipper/android/apollofrontpage/PostListFragment.kt

```diff
@@ -5,21 +5,20 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
-import androidx.fragment.app.viewModels
 import androidx.lifecycle.Observer
 import androidx.recyclerview.widget.DividerItemDecoration
 import com.quipper.android.apollofrontpage.databinding.PostListFragmentBinding
 import com.quipper.android.apollofrontpage.fragment.PostDetails
+import org.koin.android.ext.android.inject
 
 class PostListFragment : Fragment(), PostListAdapter.PostListHandler {
 
     companion object {
-
         fun newInstance() = PostListFragment()
     }
 
     private lateinit var binding: PostListFragmentBinding
-    private val viewModel: PostListViewModel by viewModels()
+    private val viewModel: PostListViewModel by inject()
```

Comment:
We should use `by viewModel()` as following the doc https://start.insert-koin.io/#/quickstart/android-viewmodel

> The by `viewModel()` function allows us to retrieve a ViewModel instance from Koin, linked to the Android ViewModelFactory.

Reply:
Resolved! I tried that earlier but somehow AS autocomplete was not showing that method. Seems we need to explicitly import that.